Week2/homework/map-filter.js

```diff
@@ -1,8 +1,7 @@
 'use strict';
 
 function doubleOddNumbers(numbers) {
-  // Replace this comment and the next line with your code
-  console.log(numbers);
+  return numbers.filter(num => num % 2 !== 0).map(num => num * 2);
```

Comment:
Just FYI - you can also use arrow functions here to replace the function definition using `function` keyword with function expressions i.e
```
const doubleOddNumbers = numbers => numbers
  .filter(num => num % 2 !== 0)
  .map(num => num * 2);
```